src/components/GetReport.jsx

```diff
@@ -23,19 +22,22 @@ const GetReport = (props) => {
 
 	const checkTrackingNumber = () =>{
 		const { trackingNumber } = props.values;
-	
-		if(RegExp(/^(ACCMP)/i).test(trackingNumber)){
+		var animalVendorComplaint = RegExp(/^(ACCMP)/i);
+		var standardVendorComplaint = RegExp(/^(CC|CRH|CS|PP|TS|CE|CP|CB|CG)\d+$/i);
+		var standardComplaint = RegExp(/^\d+$/i);
```

Comment:
what is a standardComplaint? one that doesn't map to citysourced?

Reply:
Yeah so this is where I was kind of not sure the best naming for these checks. The first two are Accela Animal and Accela common (meaning all others) and the final one is CitySourced. But since they are all vendor related and we are trying to keep away from naming by vendor in case we change I just pulled those names out of my ass. If you can think of better ones please do.

Reply:
im not sure I can come up with better names, this probably make sense, i just was curious.